compiler/src/dmd/dsymbolsem.d

```diff
@@ -2601,48 +2601,49 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
         sc.pop();
     }
 
-    override void visit(StaticCtorDeclaration scd)
+    void visitStaticCDtorDeclaration(FuncDeclaration sd, bool isDestructor)
```

Comment:
Maybe a better name for this function would be "statiCtorDtorsemantic"? In the name you've chosen it's easy to miss the C or the D.

Reply:
it should be obvious it is not like the other visits: it is not `override` and it is not named `visit`